mcsema/BC/Segment.cpp

```diff
@@ -74,7 +74,7 @@ static bool IsZero(const std::string &data) {
 // direct LLVM to put this data into the `.bss` segment.
 static bool IsZero(const NativeSegment *cfg_seg) {
   for (const auto &cfg_seg_entry : cfg_seg->entries) {
-    auto entry = cfg_seg_entry.second;
+    auto &entry = cfg_seg_entry.second;
     if (entry.blob) {
```

Comment:
Next if-else can be written more cleanly.

```
if (!entry.blob) {
  return false;
} else (!IsZero(...)) {
  return false
}
```

Reply:
Indeed, or even `!entry.blob || !IsZero(...)`.